modules/user/user.service.js

```diff
@@ -242,23 +245,26 @@ class UserService {
       ],
     });
     const savedUser = await user.save();
+
     const token = await generateToken(savedUser._id, savedUser.email, {
       expiresIn: process.env.RECOVERY_EXPIRE,
       secret: process.env.CONFIRMATION_SECRET,
     });
+
     savedUser.confirmationToken = token;
+
     await savedUser.save();
-    const message = {
-      from: process.env.MAIL_USER,
-      to: savedUser.email,
-      subject: '[HORONDI] Email confirmation',
-      html: confirmationMessage(firstName, token, language),
-    };
 
-    if (process.env.NODE_ENV !== 'test') {
-      await sendEmail(message);
-    }
+    await confirmationEmailService.confirmEmail(
```

Comment:
I think it is better to just rename it to the `EmailService` and define there one general method `sendEmail` and design it in a way that anyone who uses `EmailService` has to provide the object with all of the fields required for sending the email

Reply:
solved